tf-dataflow/README.md

```diff
@@ -0,0 +1,277 @@
+# Object Detection Model Training With Google AI Platform
+
+## When should you opt for this model training method?
+
+This model training method is applicable if you generated your data from Unity Simulation, want to use Tensorflow and you are either already using GCP or don't mind signing up. **Please note that this will incur charges for using GCP.**
+
+The workflow here will take place in 3 phases.
+
+* [Phase 1](#phase-1-get-synthetic-data-into-your-gcp-project) is where we'll transfer the data from Unity Simulation to our own GCP project's storage bucket.
+* In [Phase 2](#phase-2-transform-data-to-tensorflow-format), we'll take this data and transform it into tfrecord format using GCP's DataFlow service. We do this in order to be able to use a built-in model training algorithm on Google AI Platform.
+* Finally, in [Phase 3](#phase-3-use-google-ai-platform-to-train-an-object-detection-model), we'll submit a model training job to Google AI Platform for the data we transformed in phase 2.
+
+Here is a visual representation of the steps we just outlined:
+
+![Workflow](cloudml_workflow.svg)
+
+## Prerequisites
+
+* Clone this repo locally and navigate to the `tf-dataflow` directory.
+* A Google Cloud Platform (GCP) account
+   * Download `gcloud` tool locally - [docs](https://cloud.google.com/sdk/gcloud#downloading_the_gcloud_command-line_tool). This includes both `gcloud` and `gsutil`, which we'll be using throughout this tutorial.
+* A dataset generated on Unity Simulation - you can find out more in the [SynthDet documentation](https://github.com/Unity-Technologies/SynthDet/blob/master/docs/RunningSynthDetCloud.md).
+* Unity Simulation CLI - download and usage instructions [here](https://github.com/Unity-Technologies/com.unity.perception/blob/master/com.unity.perception/Documentation%7E/Tutorial/Phase3.md#step-3). Please carefully read this section!
+* DataFlow must be enabled for your project. More instructions can be found on Google's [official DataFlow documentation](https://cloud.google.com/dataflow/docs/quickstarts/quickstart-python). If you're an owner of the project, you'll only need to follow the guide up to Step 3 of the "Before you begin" section.
+* AI Platform must enabled for your Google Project. [Official guide here](https://cloud.google.com/ai-platform/training/docs/algorithms/object-detection-start). Please ensure you've done everything up to the "Setup" section of that guide.
```

Comment:
Is this still the case?
Also, Google Project or GCP Project?

Reply:
Same here.